src/main/java/seedu/address/model/Model.java

```diff
@@ -193,52 +193,71 @@
     void appendCommandHistoryEntry(String commandText);
 
     /**
-     * Returns true if a issue with the same identity as {@code issue} exists in the address book.
+     * Returns true if a issue with the same identity as {@code issue} exists in SunRez.
+     *
+     * @param issue Issue to be checked.
+     * @return {@code True} if issue is in SunRez.
+     * @throws NullPointerException If {@code issue} is null.
      */
     boolean hasIssue(Issue issue);
 
     /**
      * Deletes the given issue.
      * The issue must exist in SunRez.
+     *
+     * @param target Target issue to be deleted.
+     * @throws NullPointerException If {@code target} is null.
      */
     void deleteIssue(Issue target);
 
     /**
      * Adds the given issue.
+     *
+     * @param issue Issue to be added.
+     * @throws NullPointerException If {@code issue} is null.
      */
     void addIssue(Issue issue);
 
     /**
      * Replaces the given Issue {@code target} with {@code editedIssue}.
-     * {@code target} must exist in the address book.
+     * {@code target} must exist in the SunRez.
+     *
+     * @param target      Target issue to be replaced.
+     * @param editedIssue Edited Issue to replace {@code target} with.
+     * @throws NullPointerException If {@code target} or {@editIssue} is null.
      */
     void setIssue(Issue target, Issue editedIssue);
 
-
     /**
      * Closes the given issue.
      * The issue must exist in SunRez.
+     *
+     * @param target Target issue to be replaced
+     * @throws NullPointerException If {@code target} is null.
      */
     void closeIssue(Issue target);
 
     /**
-     * Checks if any issues have the given room associated with it
+     * Checks if any issues have the given room associated with it.
      *
      * @param target Room to check if it has issues associated with it.
-     * @return True if there are issues with the given room associated with it.
+     * @return {@code True} if there are issues with the given room associated with it.
      */
     boolean issuesContainRoom(Room target);
 
     /**
      * Returns an unmodifiable view of the filtered issue list.
+     *
+     * @return ObservableList of issues.
      */
     ObservableList<Issue> getFilteredIssueList();
 
     /**
-     * Updates the filter of the filtered issue list to filter by the given.
+     * Updates the filter of the filtered issue list to filter by the given
```

Comment:
Missing punctuation

Reply:
full stop is next line after `predicate`